src/LabLog/Controllers/AdminController.cs

```diff
@@ -67,9 +67,24 @@ public IActionResult AddRoom(RoomModel room)
             
         }
 
-        public IActionResult Room()
+        [Route("Room/{id}/{name?}")]
+        public IActionResult Room(Guid id, string name)
         {
-            return View();
+            string message = String.Format("{0}::{1}", id, name);

```

Comment:
use "string interpolation" here:
    string message = $"{id}::{name}";
and then consider not using a variable here at all and just using it directly